apps/quickstart-integrations-monday-code/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "monday-code-nodejs-quickstart",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "NODE_ENV=production node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon index.js"
+  },
+  "author": "Dipro Bhowmik",
+  "license": "ISC",
+  "dependencies": {
+    "@mondaycom/apps-sdk": "^0.1.3",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "jsonwebtoken": "^9.0.1",
+    "monday-sdk-js": "^0.4.9",
+    "nodemon": "^3.0.1"
```

Comment:
please move to devDependencies

Reply:
sure!

Reply:
Moving nodemon to dev-dependencies. jsonwebtoken, express, and monday-sdk-js are all used by application code. dotenv is in a grey area; since it's referenced in the source code i'm going to keep it in there.